unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/DebugController/DebugController.cs

```diff
@@ -13,11 +12,13 @@ public interface IDebugController : IDisposable
         void ShowFPSPanel();
         void SetSceneDebugPanel();
         void SetEngineDebugPanel();
+        void RunPerformanceMeterTool(float durationInMilliseconds);
     }
 
     public class DebugController : IDebugController
     {
         private DebugConfig debugConfig => DataStore.i.debugConfig;
+        private PerformanceMeterController performanceMeterController;
```

Comment:
```suggestion
        private readonly PerformanceMeterController performanceMeterController = new PerformanceMeterController();
```

With this change couldn't we avoid relying in these null checks for `performanceMeterController`?

Reply:
Totally, thx, done!